zapisy/apps/defect/models.py

```diff
@@ -1,6 +1,10 @@
 from django.db import models
 from django.contrib.auth.models import User
 from django.urls import reverse
+from gdstorage.storage import GoogleDriveStorage
+
+# Define Google Drive Storage
+gd_storage = GoogleDriveStorage()
```

Comment:
Czy to jest na pewno dobre miejsce na to? Spodziewałbym się bardziej osobnego pliku. Zwłaszcza, że nie jest to użyte w tym.

Reply:
jest użyte :D